diff -urN nestopia-1.43/Makefile nestopia-1.43-patched/Makefile
--- nestopia-1.43/Makefile	2012-11-28 21:23:14.000000000 +0000
+++ nestopia-1.43-patched/Makefile	2012-12-05 00:48:58.162598770 +0000
@@ -26,9 +26,9 @@
 LDFLAGS += -Wl,--as-needed
 
 EXE  = nestopia
-LIBS = -lstdc++ -lm -lz -lasound $(shell sdl-config --libs) $(shell pkg-config --libs gtk+-3.0)
+LIBS = -lstdc++ -lm -lz -lasound -lX11 $(shell sdl-config --libs) $(shell pkg-config --libs gtk+-3.0)
 
-PREFIX = /usr/local
+PREFIX = /usr
 BINDIR = $(PREFIX)/bin
 DATADIR = $(PREFIX)/share/nestopia
 
